lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -133,6 +142,16 @@ class_exists(IgnoreAnnotation::class);
         $this->phpParser = new PhpParser();
     }
 
+    /**
+     * Add a new annotation to the ignored annotation namespaces with regard to exception handling.
+     *
+     * @param string $namespace
+     */
+    public function addIgnoredNamespace($namespace)
```

Comment:
This is a new method. As such it can use type declarations, and we should IMO mark it as `final`.

```suggestion
    final public function addIgnoredNamespace(string $namespace): void
```